src/mongocxx/test/collection.cpp

```diff
@@ -534,21 +534,25 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
 
         // copy into a vector.
         std::vector<bsoncxx::document::value> results;
+        
         for (auto&& result : distinct_results) {
```

Comment:
An exception is being thrown at this line on travis. I didn't change the output of collection::distinct, and I'm not able to reproduce the exception locally. Does anyone have an idea of why this might be occurring?